README.md

```diff
@@ -1,4 +1,4 @@
-# Acme Gifts
+# Acme Gifts [![Build Status](https://travis-ci.org/OpenLiberty/sample-acmegifts.svg?branch=master)](https://travis-ci.org/OpenLiberty/sample-acmegifts)
```

Comment:
Is this "[![Build Status](https://travis-ci.org/OpenLiberty/sample-acmegifts.svg?branch=master)](https://travis-ci.org/OpenLiberty/sample-acmegifts)"  correct?

Reply:
yes. it's marked unknown since the master branch on OpenLiberty/sample-acmegifts hasn't taken in the travis.yml file yet. I was using my own fork as a placeholder until Cindy noticed it should be ready for OpenLiberty when we merge.